oracle/src/tx/sendTx.js

```diff
@@ -2,6 +2,8 @@ const Web3Utils = require('web3-utils')
 const fetch = require('node-fetch')
 const rpcUrlsManager = require('../services/getRpcUrlsManager')
 
+const { ORACLE_TX_REDUNDANCY } = process.env
```

Comment:
Please update the documentation files related to the oracle with this parameter. Please also check that it will be propagated to the target system automatically by the ansible deployment scripts.

Reply:
I have updated corresponding docs.
Regarding Ansible deployment, if I am not mistaken, it should be enough to add `ORACLE_TX_REDUNDANCY` in `deployment/roles/oracle/templates/.env.j2`.